$OpenBSD: patch-setup_py,v 1.1.1.1 2009/09/22 15:09:39 jasper Exp $

Force usage of setuptools over distutils to prevent an:
error: option --single-version-externally-managed not recognized

And remove empty directories.

--- setup.py.orig	Tue Sep 22 14:58:42 2009
+++ setup.py	Tue Sep 22 15:03:48 2009
@@ -14,6 +14,7 @@ simpler.
 
 import commands
 from distutils.extension import Extension
+import setuptools.command.install
 import sys
 
 from kiwi import kiwi_version
@@ -57,12 +58,6 @@ setup(name="kiwi",
                              'kiwiwidgets', '*.png')),
                   (get_site_packages_dir('gazpacho', 'widgets'),
                    listfiles('gazpacho-plugin', 'kiwiwidgets.py')),
-                  ('share/doc/kiwi',
-                   ('AUTHORS', 'ChangeLog', 'NEWS', 'README')),
-                  ('share/doc/kiwi/howto',
-                   listfiles('doc/howto/', '*')),
-                  ('share/doc/kiwi/api',
-                   listfiles('doc/api/', '*')),
                   ],
       scripts=['bin/kiwi-i18n',
                'bin/kiwi-ui-test'],
